app/Example.java

```diff
@@ -0,0 +1,50 @@
+import android.view.View;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Message;
+import io.ably.lib.types.Param;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
+
+	public void publishAndSubcscribe(Button buttonPublish, Button buttonHistory) {
+		AblyRealtime realtime = new AblyRealtime(API_KEY);
+        /* Get a channel in the "persisted" namespace */
+        final Channel channel = realtime.channels.get("persisted:sounds");
```

Comment:
Spacing looks odd, looks like you have tabs for the `public void` definition and then just one space here?  We need consistent 2 char spacing or tabs, but not a mixure